boa/contracts/vyper/vyper_contract.py

```diff
@@ -163,6 +181,14 @@ def abi(self):
     def _constants(self):
         return ConstantsModel(self.compiler_data)
 
+    def verify(self, explorer, license_type: str = None) -> None:
```

Comment:
does it make sense to default to the global verifier (set with `set_blockscout`)?

Reply:
Then `set_explorer` so we can expand to different ones?